packages/emile/emile.0.7/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name:         "emile"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/emile"
+bug-reports:  "https://github.com/dinosaure/emile/issues"
+dev-repo:     "git+https://github.com/dinosaure/emile.git"
+doc:          "https://dinosaure.github.io/emile/"
+license:      "MIT"
+synopsis:     "Parser of email address according RFC 822"
+description:  """Parser of email address according RFC 822"""
+
+build: ["dune" "build" "-p" name "-j" jobs]
+run-test: ["dune" "runtest" "-p" name "-j" jobs]
+
+depends: [
+  "ocaml"    {>= "4.03.0"}
+  "dune"     {build}
```

Comment:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency. Please return the fix upstream this time